.editorconfig

```diff
@@ -22,10 +22,6 @@ indent_size = 4
 insert_final_newline = true
 max_line_length = 120
 
-dotnet_style_qualification_for_field = true:warning
```

Comment:
There is no usage of `this` keyword, so I removed those settings.

Reply:
Good catch. (I had simply copied a number of configs over from some new projects... but didn't really align these with the existing code.) To continue to provide the hints but stay consistent with the existing code, this could be changed to:

```diff
-dotnet_style_qualification_for_field = true:warning
-dotnet_style_qualification_for_property = true:warning
-dotnet_style_qualification_for_method = true:warning
-dotnet_style_qualification_for_event = true:warning
+dotnet_style_qualification_for_field = false:warning
+dotnet_style_qualification_for_property = false:warning
+dotnet_style_qualification_for_method = false:warning
+dotnet_style_qualification_for_event = false:warning
```

Reply:
Done